cassiopeia/src/main/java/org/apache/tika/parser/ocr/TesseractOCRParser.java

```diff
@@ -350,4 +345,38 @@ static String getTesseractProg() {
         return System.getProperty("os.name").startsWith("Windows") ? "tesseract.exe" : "tesseract";
     }
 
+    /**
+     * This function gets the list of available languages of Tesseract.
+     * 
+     * @param propertyManager
+     * @return String[] - the list of available languages
+     */
+    public List<String> getTessLangs(IPropertiesManager propertyManager) {
+        String tesseractBin = propertyManager.getProperty(Properties.TESSERACT_BIN_FOLDER);
+        String command = tesseractBin + "/tesseract --list-langs";
+        List<String> output = new ArrayList<String>();
+        BufferedReader reader = null;
+        try {
+            Process proc = Runtime.getRuntime().exec(command);
+            reader = new BufferedReader(new InputStreamReader(proc.getInputStream()));
+            String line = "";
+            while ((line = reader.readLine()) != null) {
+                output.add(line);
+            }
```

Comment:
you still need a comment here, explaining why you are doing what you're doing. and wasn't the first line, not a language?

Reply:
I will add the comments.
The first line is "List of available languages:(3)" and therefore it is not required.